tools/rosbag_storage/include/rosbag/bag.h

```diff
@@ -569,7 +581,7 @@ void Bag::doWrite(std::string const& topic, ros::Time const& time, T const& msg,
 
         // Check if we want to stop this chunk
         uint32_t chunk_size = getChunkOffset();
-        logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
+        CONSOLE_BRIDGE_logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
```

Comment:
This patch is targeting `lunar-devel` and there is no reason to backport this into Indigo. Therefore I don't think the second commit is necessary.